enki/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ if (CMAKE_HOST_UNIX)
 	add_definitions("-fPIC")
 endif (CMAKE_HOST_UNIX)
 
+if (MSVC)
+	add_definitions(-DWIN32)
```

Comment:
This is your re: your comment from e50f6958049d79cbde2a94387edcd9c5870ce403, which was never resolved because I didn't have the time to check the defines on all platforms.

In this case the problem is compilation using MS Visual Studio. If you remove this clause you will need to change the `#ifdef` in `enki/Geometry.h`.